src/snmp_core.cc

```diff
@@ -389,7 +389,7 @@ snmpDecodePacket(SnmpRequest * rq)
         return;
     }
 
-    debugs(49, 5, HERE << "Called.");
+    debugs(49, 5, "Called.");
```

Comment:
Agreed, but this PR automatically removes HERE, and nothing else should be required.

As for the standardization, I was thinking about printing a semicolon instead of an empty string. It would kind of address the "hanging `:`" problem. Would you like that better?

Removing trailing `:` would be better, but to do that, we would have to split debugs() macro in two or three parts and introduce a special macro for these "called" or "reached" cases:

```
debugHere(79, 5);
```

I do not think the resulting complexity is worth it. The debugs() macro is already very complex and difficult to work with. What do you think?